oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/config/annotation/web/configurers/OidcClientRegistrationTests.java

```diff
@@ -418,8 +418,9 @@ public void requestWhenClientRegistersWithCustomMetadataThenSavedToRegisteredCli
 				.grantType(AuthorizationGrantType.CLIENT_CREDENTIALS.getValue())
 				.scope("scope1")
 				.scope("scope2")
-				.claim("custom-metadata-name-1", "value-1")
-				.claim("custom-metadata-name-2", "value-2")
+				.claim("custom-metadata-1", "value-1")
```

Comment:
I'm not sure if we need to change the existing `custom-metadata-name-1` and `custom-metadata-name-2`? I would prefer minimal changes possible. Can we reuse existing custom claims input?